### PR TITLE
[DOC] add landing page profile for github organization

### PR DESCRIPTION
**Pull request by Remi-Gau**

### CODE REVIEW

**Review by bendhouseart — APPROVED** [review]
Oh [this](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/about-your-profile) is neat! Yeah merge it!

**Review by sappelhoff — APPROVED** [review]

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# 👋 Welcome to BIDS github organization 👋
+
+Click the following links if you want to
+
+- [read the BIDS specification](https://bids-specification.readthedocs.io/en/latest/)
+- [validate a BIDS dataset](https://bids-standard.github.io/bids-validator/)
+- [want to get started with BIDS](https://bids-standard.github.io/bids-starter-kit/)
+- [find BIDS datasets examples](https://github.com/bids-standard/bids-examples#bids-examples)
+- need tools to work with BIDS dataset for
+  - [python](https://bids-standard.github.io/pybids/)
```
Comment (arokem) [inline profile/README.md]:
```suggestion
  - [Python](https://bids-standard.github.io/pybids/)
```

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# 👋 Welcome to BIDS github organization 👋
+
+Click the following links if you want to
+
+- [read the BIDS specification](https://bids-specification.readthedocs.io/en/latest/)
+- [validate a BIDS dataset](https://bids-standard.github.io/bids-validator/)
+- [want to get started with BIDS](https://bids-standard.github.io/bids-starter-kit/)
```
Comment (arokem) [inline profile/README.md]: Repeats "want to":

```suggestion
- [get started with BIDS](https://bids-standard.github.io/bids-starter-kit/)
```

File: profile/README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+- [find BIDS datasets examples](https://github.com/bids-standard/bids-examples#bids-examples)
+- need tools to work with BIDS dataset for
```
Comment (arokem) [inline profile/README.md]:
```suggestion
- work with BIDS dataset in
```

**Review by arokem — APPROVED** [review]
Thanks for doing this! Some small suggestions from me.

**Comment by robertoostenveld** [comment]
it might be too long, but let's give it a try.

**Comment by Remi-Gau** [comment]
> it might be too long, but let's give it a try.

yeah I was afraid of that too, but I thought "better than nothing"

File: profile/README.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+- [read the BIDS specification](https://bids-specification.readthedocs.io/en/latest/)
+- [validate a BIDS dataset](https://bids-standard.github.io/bids-validator/)
+- [get started with BIDS](https://bids-standard.github.io/bids-starter-kit/)
+- [find BIDS datasets examples](https://github.com/bids-standard/bids-examples#bids-examples)
+- work with BIDS dataset in
+  - [Python](https://bids-standard.github.io/pybids/)
+  - [MATLAB / Octave](https://bids-matlab.readthedocs.io/en/latest/index.html)
+  <!-- - R -->
+- [find a list of BIDS converters](https://bids.neuroimaging.io/benefits.html#converters)
+- [get information on how to submit a BIDS extension proposal](https://docs.google.com/document/d/1pWmEEY-1-WuwBPNy5tDAxVJYQ9Een4hZJM06tQZg8X4/edit)
```
Comment (guiomar) [inline profile/README.md]:
```suggestion
- [list of BIDS extension proposals](https://bids.neuroimaging.io/get_involved.html)
- [get information on how to submit a new BIDS extension proposal](https://docs.google.com/document/d/1pWmEEY-1-WuwBPNy5tDAxVJYQ9Een4hZJM06tQZg8X4/edit)
```

**Comment by guiomar** [comment]
It looks very good to me! Thank you :)

**Comment by robertoostenveld** [comment]
looks nice and makes our github repo a more friendly place for first-time visitors!

**Pull request merged**